### DISCUSSION TITLE
seeing nginx default landing page on mitm.it

### DISCUSSION THREAD

**Discussion by smudgesmudge32** [mitmproxy/mitmproxy#6232] [General]
After adding in the mitm.it proxy in my IOS device with the correct settings I'm now being served an nginx page rather than the profile/certificate installation. Can anyone please take a look at this? Alternatively, is there a way to download these certs and install them as an IOS profile before hand (using other tools), or is this unique to every device?

**Comment by mhils** [comment]
The [mitm.it](http://mitm.it/) page is served from your local mitmproxy instance (when proxied). And we don't have nginx in there, so this is a problem with your configuration and not with mitmproxy.

You also find your local certs in `~/.mitmproxy` if that helps.

**Comment by smudgesmudge32** [comment]
That does help! A classic case of PEBKAC :)

Thanks and good work
